packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -554,17 +564,14 @@ class _PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
 
   @override
   Widget build(BuildContext context) {
-    if (widget.child == null) {
-      return new IconButton(
-        icon: _getIcon(Theme.of(context).platform),
-        padding: widget.padding,
-        tooltip: widget.tooltip,
-        onPressed: showButtonMenu,
-      );
-    }
-    return new InkWell(
-      onTap: showButtonMenu,
-      child: widget.child,
-    );
+    return widget.child != null
+      ? new InkWell(
+          onTap: showButtonMenu,
+          child: widget.child)
+      : new IconButton(
+          icon: widget.icon ?? _getIcon(Theme.of(context).platform),
+          padding: widget.padding,
+          tooltip: widget.tooltip,
+          onPressed: showButtonMenu);
```

Comment:
you need some extra newlines (before the `)`s) and some trailing commas (on those last lines) to match the prevailing style.